libraries/AP_WindVane/AP_WindVane.h

```diff
@@ -60,15 +60,15 @@ class AP_WindVane
     static AP_WindVane *_s_instance;
 
     // Wind Vane parameters
-    AP_Int8  _type;             // type of windvane being used
-    AP_Int8  _pin;              // analog or pwm pin connected to sensor
-    AP_Float _analog_volt_low;  // voltage when windvane is pointing at 0 degrees
-    AP_Float _analog_volt_high; // voltage when windvane is pointing at 359 degrees
-    AP_Int16 _pwm_low;          // PWM when 'windvane' is pointing at 0 degrees
-    AP_Int16 _pwm_value;        // PWM when 'windvane' is pointing at 359 degrees
+    AP_Int8 _WindVane_Type;             // type of windvane being used
```

Comment:
no need to prepend _WindVane.  We are already inside the AP_WindVane class... also we normally don't use camel case but instead small letters with underscores..